### PR TITLE
PROJECT 2 : MILESTONE TWO

### PR DESCRIPTION
**Pull request by tanzila-abedin**
I have forked this project from my previous working partner with whom I have completed **milestone-one,** of this project
This is the other half of the **project, milestone-two.** My current coding partner @JulianaOsemeke and I are working on it.

### CODE REVIEW

**Comment by tanzila-abedin** [comment]
Hello 

@JulianaOsemeke and my project repository do contain GitHub flow action but it refuses to apply to the pull request to check for linter errors.
We are failing to understand what might be causing it. Kindly advise!

Thank you :)

File: index.html [hunk 1]
```diff
@@ -0,0 +1,671 @@
...
+                <h1><a href="#">Hard Fork</a></h1>
+            </div>
+            <!-- <div class="cat-gradient"> -->
+            <div class="cat-img-box">
+              <img src="images/categories/hardfork.png" alt="game stop">
+            </div>
+            <div class="cat-img-txt">
+              <h4>MOST POPULAR</h4>
+              <P><a href="#" style="font-weight: bold; font-size: 20px; color: white">GameStop stocks hit record high — thanks to Redditors</a></P>
+              <p><a class="cat-grey-txt"href="#">Ivan Mehta</a><span class="grey-txt">. 1 day ago</span></p>
```
Comment (yokoworks) [inline index.html]: - [x] Use lowercase element names: you can see specific description [here](https://www.w3schools.com/html/html5_syntax.asp)
- [x] Using inline styles is one of the [common mistakes](https://speckyboy.com/good-bad-css-practices). Styles should be defined as separated from html.

**Review by yokoworks — CHANGES_REQUESTED** [review]
# Status: Changes Required♻️ 

### Great job so far, team👍  

You two have made an amazing job🚀
There are some points to improve before you go ahead.
Let's cover a few details together to make your project better😊 

## Required changes:
- [x] Use meaningful names for your Pull Requests and add a summary of its content as described [here](https://microverse.zendesk.com/hc/en-us/articles/360055660814-Professional-GitHub-repository)
- [x] Add a descriptive README file to your repositories as described [here](https://microverse.zendesk.com/hc/en-us/articles/360055660814-Professional-GitHub-repository)
- [x] CSS styles would use media queries min-width: "1024px" as the large breakpoint, instead of "1025px"
- [x] Live demo link is not the latest. Could you refresh it to the latest one?

### Latest news section

- [x] It has only 3 rows, but it would have 4 articles per row in Desktop view
- [x] Images should be the full width of the container in tablet and desktop views

### Categories (Apps, Gear, Tech, etc) section  in each category

- [x] The first article title (eg. "GameStop stocks hit record high — thanks to Redditors") should be a header(hx) tag and a link (a tag)
- [x] The second article title (eg. "Alibaba shares jump 8% as Jack Ma surfaces after 3 months") should use a header(hx) tag and a link (a tag)
- [x] In the rest of the articles, each list should have each link (a tag). Now link tags are included in link(a tag), but link tag(ul/li) and link(a tag) should be used like below.
eg. ` <ul><li><a href="">Article title</a></li></ul>`

### Latest Deals section

- [x] The link (a tag) to "See all deals" is in the same row that the title but aligned to the right
- [x] Each article should have a title (hx tag), both of them with a link (a tag)
- [x] There should be 3 articles per row on a tablet view, and 4 on desktop
- [x] Images should be the full width of the container in tablet and desktop views
- [x] This section should have a little margin or padding to the left side of it

## Optional changes [OPTIONAL]
- [x] [OPTIONAL] You can use `object-fit: cover;` in each image in order to maintain the ratio of images

Please request another review once you are ready👍


### With all my respect 🍀

Yoko Saka

**Comment by tanzila-abedin** [comment]
@yocosaka  Good Day!

We have made the changes you asked for. Kindly have a look!

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light: 

Hi, @tanzila-abedin and @JulianaOsemeke, great job on this project :+1: However, I would like to point out some of the missing comments from the previous review which were not addressed. Kindly check the comments below:

- `Latest news` (min-width: 768px)
`Original`
w
`Clone`
q
- `Categories section` (min-width: 768px)
Elements are to be aligned in 1 column but currently, we have elements aligned in 2 columns. Let us kindy fix this. Kindly compare our clone project to the [original](https://perma.cc/M5ZV-Q2D6) on all screen sizes
`Original`
e
`Clone`
r
- `Latest deals` (min-width: 768px)
`Original`
t
`Clone`
y
- README file: In the author section, one of the links to the social handles is not working it is broken. Also, it is not professional to have an Instagram account where it is also a private account, kindly use Twitter instead, where we have our professional profile and it is public

In all the app is not responsive, let us take it section by section and compare our app with the original. Implement the responsive requirements :smile: 

Once this is addressed, kindly submit another code review request :+1: 
_Happy coding_ :woman_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Comment by tanzila-abedin** [comment]
Hello @addod19 

When we were cloning this website we had followed the dimensions of **Mobile (min-width: 320px ),Tablet (768px) and Laptop(1024px)** given by the Microverse project scope. For each view, they wanted us to style as shown below and not as per the original website.
**Microverse Scope:**
Screenshot from 2021-02-27 15-19-56
Screenshot from 2021-02-27 15-20-01
Screenshot from 2021-02-27 15-20-09

**Here's the SS of the website we cloned:**
Mobile view
Screenshot from 2021-02-27 15-14-41
Tablet view 
Screenshot from 2021-02-27 15-13-51
Laptop view 
Screenshot from 2021-02-27 15-15-36

**Review by voscarmv — APPROVED** [review]
# Milestone status: Approved ✨ 😎👍 💯 ✔️
### You may merge this branch and move forward ⛙🛣

**[OPTIONAL]**

- The only detail I could notice is that some of your images appear stretched on different display modes. Please take some time to correct this issue:
image


### Questions or comments? Reach me on [Twitter](https://twitter.com/voscarmv) or [LinkedIn](https://linkedin.com/in/oscar-mier) 🧑‍💻🗨️🧑‍💻 
# Happy Hacking! 🧠 ⌨️ 🖱️ 🖥️

**Pull request merged**